config/default.json

```diff
@@ -1,4 +0,0 @@
-{
-  "db": "mongodb://localhost/goodpoints",
```

Comment:
yes

Reply:
for heroku deployment

Reply:
it's needed for anyone who will clone the project to local machine also. This file shows you what are the variables so that you can overwrite them with ENV variables.

Reply:
ok